src/ompl/geometric/planners/bitstar/datastructures/src/IntegratedQueue.cpp

```diff
@@ -1578,7 +1578,7 @@ namespace ompl
 
         bool BITstar::IntegratedQueue::isCostBetterThan(const ompl::base::Cost& a, const ompl::base::Cost& b) const
         {
-            return a.value() < b.value();
+            return opt_->isCostBetterThan(a, b);
```

Comment:
To be 100% honest, I don't really know for sure, but I assume that it's for deciding when to finish and stuff like that?